script.js

```diff
@@ -1,4 +1,66 @@
+var Scissors = "scissors";
+var Paper = "paper";
+var Stone = "stone";
```

Comment:
Good move saving these string constants to variables. When doing so, best practice for naming the variables is to use screaming snake case. One word example: SCISSORS. Multiple words: RANDOM_VARIABLE.